lib/ts/recipe/dashboard/api/implementation.ts

```diff
@@ -38,7 +38,9 @@ export default function getAPIImplementation(): APIInterface {
             const authMode: AuthMode = input.options.config.authMode;
 
             if (superTokensInstance.supertokens !== undefined) {
-                connectionURI = superTokensInstance.supertokens.connectionURI;
+                connectionURI = new NormalisedURLDomain(
+                    superTokensInstance.supertokens.connectionURI.split(";")[0]
```

Comment:
i think the supertokens config is optional from a types point of view. Need to take that into account?